README.md

```diff
@@ -145,51 +225,73 @@ module.exports = function(hermione, opts) {
 }
 ```
 
-Свойства объекта hermione:
-* `config` - конфиг, который используется в раннере. Может быть модифицирован
-* `events` - список событий, на которые можно подписаться
-
-#### События
-* `RUNNER_START` - триггерится перед запуском тестов. Обработчик может вернуть промис, тесты запустятся только после того как промис зарезолвится
-* `RUNNER_END` - триггерится после завершения всех тестов. Аналогично `RUNNER_START`, обработчик может вернуть промис
-* `SUITE_BEGIN` - запустился новый сьют
-* `SUITE_END` - сьют завершен
-* `TEST_BEGIN` - тест пошел на исполнение
-* `TEST_END` - тест завершился
-* `TEST_PASS` - тест пройден
-* `TEST_FAIL` - тест упал
-* `TEST_PENDING` - тест скипнут
-* `RETRY` - тест упал, но ушел на retry
-* `ERROR` - произошла какая-то ошибка не в тесте, например не поднялся браузер
-* `INFO` - (зарезервировано)
-* `WARNING` - (зарезервировано)
-* `EXIT` - триггерится когда получен сигнал на завершение (например при нажатии на Ctrl + C). Обработчик может вернуть промис
-
-### Прочие свойства
+**Properties of `hermione` object**
+
+Property name             | Description
+------------------------- | -------------
+`config`                  | Config which is used in test runner. Can be mutated.
+`events`                  | Events list for subscribtion
+
+**Available events**
+
+Event                     | Description
+------------------------- | -------------
+`RUNNER_START`            | Will be triggered before tests execution. If hanlder return a promise, tests will be executed only after promise resolving.
+`RUNNER_END`              | Will be triggered after tests execution. If hanlder return a promise, tests will be executed only after promise resolving.
+`SUITE_BEGIN`             | Test suite is about to execute
+`SUITE_END`               | Test suite execution is finished
+`TEST_BEGIN`              | Test is about to execute
+`TEST_END`                | Test execution is finished
+`TEST_PASS`               | Test passed
+`TEST_FAIL`               | Test failed
+`TEST_PENDING`            | Test is skipped
+`RETRY`                   | Test failed but went to retry
+`ERROR`                   | Generic (no tests) errors. For instance, a browser cannot be loaded
+`INFO`                    | Reserved
+`WARNING`                 | Reserved
+`EXIT`                    | Will be triggerd when sigterm is recieved (for example, Ctrl + C). Handler can return a promise.
+
+### mochaOpts
+Extra options for `mocha`
+```javascript
+mochaOpts: {
+    retries: 3
```

Comment:
You may specify any mocha opts here. Retries may be passed too, but it would be better to use something else in example.